requirements.txt

```diff
@@ -6,7 +6,6 @@ PyYAML==5.3
 pyxdg==0.26
 requests==2.20.0
 requests_unixsocket==0.1.5
-http://archive.ubuntu.com/ubuntu/pool/main/p/python-apt/python-apt_1.6.5ubuntu0.2.tar.xz; sys_platform == 'linux'
 https://launchpad.net/python-distutils-extra/trunk/2.39/+download/python-distutils-extra-2.39.tar.gz
```

Comment:
IIRC, this dep is only required for python-apt

Reply:
That's what I figured, I was gonna cut it afterwards.